web_tree_date_search/README.rst

```diff
@@ -0,0 +1,40 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Tree Date Search
+==============================
```

Comment:
Hi @fossoult  thanks for your contribution
I will check mostly format and guidelines in your PR
Can you remove extra underline and adjust with sentence length?